DogeNet.Registration/Controllers/RegistrationController.cs

```diff
@@ -0,0 +1,40 @@
+﻿using DogeNet.BLL.Features.Account.CreateAccount;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DogeNet.Registration.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class RegistrationController : ControllerBase
+    {
+        private readonly IMediator mediator;
+
+        public RegistrationController(IMediator mediator)
+        {
+            this.mediator = mediator;
+        }
+
+        [HttpPost]
+        public async Task<IActionResult> Register(CreateAccountCommand command) 
+        {
+            // Validation here...
+            var validator = new CreateAccountValidator();
```

Comment:
Validator should be called automatically

Reply:
How?

Reply:
https://docs.fluentvalidation.net/en/latest/aspnet.html